src/current/_includes/releases/v24.1/v24.1.0-alpha.5.md

```diff
@@ -0,0 +1,120 @@
+## v24.1.0-alpha.5
+
+Release Date: April 1, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-0-alpha-5-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [Changefeeds]({% link v24.1/change-data-capture-overview.md %}) now default to evenly distributing their work across all replicas, including followers, regardless of leaseholder placement. To disable this behavior, set the [cluster setting]({% link v24.1/cluster-settings.md %}) `changefeed.balanced_distribution.enabled` to `false`. If disabled, now use BulkOracle for planning, which distributes work evenly across all replica in the locality filter, including followers if enabled. This is enabled by default with the cluster setting `changefeed.balanced_distribution.enabled`. If disabled, changefeed planning reverts to its previous behavior for distributing work. [#120077][#120077]
```

Comment:
I don't understand the sentence that starts with "If disabled, now use BulkOracle..." From how it reads, it look like you could delete the following:
```suggestion
- [Changefeeds]({% link v24.1/change-data-capture-overview.md %}) now default to evenly distributing their work across all replicas, including followers, regardless of leaseholder placement. To disable this behavior, set the [cluster setting]({% link v24.1/cluster-settings.md %}) `changefeed.balanced_distribution.enabled` to `false`. If disabled, changefeed planning reverts to its previous behavior for distributing work. [#120077][#120077]
```